service-tests/helpers/mocks.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const invalidJSON = function() {
+  return [
+    200,
+    'invalid json',
```

Comment:
Would something a little stranger here, like `'this string <here> is [definitly.not] valid json'` make it clearer that this is the json string, and not an error message that gets passed in?

Reply:
I've change it to `{{{{{invalid json}}`.